<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1133</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Photo Manager</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1121</width>
     <height>811</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>중복 사진 삭제</string>
    </attribute>
    <widget class="QPushButton" name="CmpSelecAllBtn">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>80</y>
       <width>112</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Select all</string>
     </property>
    </widget>
    <widget class="QScrollArea" name="scrollArea_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>670</y>
       <width>1111</width>
       <height>107</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1109</width>
        <height>105</height>
       </rect>
      </property>
      <widget class="QTextBrowser" name="resultText">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1111</width>
         <height>91</height>
        </rect>
       </property>
      </widget>
      <widget class="QProgressBar" name="progressBar">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>90</y>
         <width>1111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QPushButton" name="BaseSelecAllBtn">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>80</y>
       <width>112</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Select all</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cmpLoadBtn">
     <property name="geometry">
      <rect>
       <x>1080</x>
       <y>40</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>50</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Compare Path</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="comparePath">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>40</y>
       <width>381</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>531</width>
       <height>241</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>529</width>
        <height>239</height>
       </rect>
      </property>
      <widget class="QListView" name="baseListView">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>531</width>
         <height>241</height>
        </rect>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::MultiSelection</enum>
       </property>
       <property name="selectionRectVisible">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QPushButton" name="removeSelecFilesBtn">
     <property name="geometry">
      <rect>
       <x>910</x>
       <y>80</y>
       <width>201</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Remove selected files</string>
     </property>
    </widget>
    <widget class="QLabel" name="baseImgView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>370</y>
       <width>531</width>
       <height>281</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="cmpBtn">
     <property name="geometry">
      <rect>
       <x>1000</x>
       <y>0</y>
       <width>112</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Compare</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>30</y>
       <width>20</width>
       <height>621</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="basePath">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>401</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="cmpImgView">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>370</y>
       <width>551</width>
       <height>281</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="baseLoadBtn">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>40</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QScrollArea" name="scrollArea_2">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>120</y>
       <width>551</width>
       <height>241</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>549</width>
        <height>239</height>
       </rect>
      </property>
      <widget class="QListView" name="cmpListView">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>551</width>
         <height>241</height>
        </rect>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::MultiSelection</enum>
       </property>
       <property name="selectionRectVisible">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Base Path</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>스마트 폴더 구분</string>
    </attribute>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>1091</width>
       <height>721</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="foldersLayout"/>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>baseLoadBtn</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>baseLoadBtnClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>545</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>579</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmpLoadBtn</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>cmpLoadBtnClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1108</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>1024</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BaseSelecAllBtn</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>baseSelecAllBtnClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>492</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>480</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CmpSelecAllBtn</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>cmpSelecAllBtnClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>883</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>871</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>removeSelecFilesBtn</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>removeSelecFilesBtnClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>983</x>
     <y>138</y>
    </hint>
    <hint type="destinationlabel">
     <x>978</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmpBtn</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>cmpBtnClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1035</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>1048</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>baseLoadBtnClicked()</slot>
  <slot>cmpLoadBtnClicked()</slot>
  <slot>cmpBtnClicked()</slot>
  <slot>cmpSelecAllBtnClicked()</slot>
  <slot>baseSelecAllBtnClicked()</slot>
  <slot>removeSelecFilesBtnClicked()</slot>
 </slots>
</ui>
